packages/ui/src/theme/css/component/showPassword.scss

```diff
@@ -0,0 +1,6 @@
+.amplify-field__show-password {
+  &--error {
+    color: var(--amplify-components-fieldcontrol-error-border-color);
+    border-color: var(--amplify-components-fieldcontrol-error-border-color);
+  }
+}
```

Comment:
Let's include this under `passwordField.scss` since this is not a separate primitive.

Reply:
+1, please do not create a new css file